network/network_windows.go

```diff
@@ -55,6 +58,26 @@ func UseHnsV2(netNs string) (bool, error) {
 	return useHnsV2, err
 }
 
+// this hnsv2 variable is package level variable in network
+// we do this to avoid passing around os specific objects in platform agnostic code
+var hnsv2 hnswrapper.HnsV2WrapperInterface = hnswrapper.Hnsv2wrapper{}
+
+var Hnsv1 hnswrapper.HnsV1WrapperInterface = hnswrapper.Hnsv1wrapper{}
```

Comment:
Unfortunate that hnsv1 would need to exported here, its used by cni/network in the consecutive add

Reply:
you can use hnsendpoitclient interface intialized in netplugin and use it as like before?

Reply:
Closing this comment out per our discussion offline, to use hnsendpoitclient would require some non trivial refactor between cni/network and root network package.